test/plugins/LabelBot/label_bot.spec.ts

```diff
@@ -33,7 +33,7 @@ describe("LabelBotPlugin", () => {
       ],
     });
     assert.deepEqual(setLabels, {
-      labels: ["integration: mqtt", "merging-to-master", "core"],
+      labels: ["integration: mqtdt", "merging-to-master", "core"],
```

Comment:
What are we testing? Should it be a missing integration?

Reply:
uppppps, that should not be there.